### PR TITLE
l11 document object model

### PR DESCRIPTION
**Pull request by Angelbt12**

### CODE REVIEW

File: src/ab/js/l11/The-cat’s-hat.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<style>
+    body { min-height: 200px; }
+
+  </style>
+  
+  <img src="img/cat.png" id="cat" style="position: absolute;">
```
Comment (carin91) [inline src/ab/js/l11/The-cat’s-hat.html:6]: you need to include the images into the branch

File: src/ab/js/l11/Elements-by-tag-name.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    function byTagName(node, tagName) {
+  tagName = tagName.toLowerCase();
+
+  let result = [];
+
+  function findNodes(currentNode) {
+    for (let i = 0; i < currentNode.childNodes.length; i++) {
+      const childNode = currentNode.childNodes[i];
+      if (childNode.nodeType === 1 && childNode.nodeName.toLowerCase() === tagName) {
+      }
```
Comment (carin91) [inline src/ab/js/l11/Elements-by-tag-name.html:14]: you are returning a void array, i guess do you forgot to commit the definitions inside this scope.

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/ab/js/l11/The-cat’s-hat.html (hunk 1 above)
Comment (Angelbt12) [reply src/ab/js/l11/The-cat’s-hat.html:6]: ok

File: src/ab/js/l11/Elements-by-tag-name.html (hunk 2 above)
Comment (Angelbt12) [reply src/ab/js/l11/Elements-by-tag-name.html:14]: yes sorry I'll fix it

**Pull request merged**